devmaster/modules/devshop/devshop_projects/inc/create/create.inc

```diff
@@ -57,7 +57,7 @@ function devshop_projects_create_wizard($step = NULL) {
   else {
     // Load latest project object from database and use that for the ctools cache.
     $project_node = node_load($project->nid);
-    $project = $project_node->project;
+    $project = $project_node->project ?? new stdClass();
```

Comment:
Why would $project->nid not work? 
  - Because the project node was deleted?
 - Should we catch that edge case and alert the user? If the project cache NID refers to a node that no longer exists, we should explain to the user.

Reply:
I don't know. I received run time errors and added the null coalescing operator, which avoided them.